includes/admin/class-wcmypa-admin.php

```diff
@@ -1231,7 +1231,8 @@ public static function getExtraOptionsFromOrder(WC_Order $order): array
      */
     private function printDeliveryDate(DeliveryOptions $deliveryOptions): void
     {
-        if (null !== $deliveryOptions->getDate()) {
+        if ("" !== $deliveryOptions->getDate() || AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME ===
```

Comment:
Moet dit niet iets overzichtelijker volgens de wiki regels? ik dacht zoiets maar check t even...:
``` 
        if (
          "" !== $deliveryOptions->getDate()
          || AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME ===  $deliveryOptions->getPackageType()
        ) {

Reply:
Waarom een lege string? Dat is wmb illegaal

Reply:
Waarom een lege string? Dat is zwaar illegaal. Doe gewoon `if ($deliveryOptions->getDate() ...)`, dat is stukken duidelijker en werkt op alle falsy waarden. Je kunt ook nog die package type check in een variabele zetten. Zo wordt het ineens heel leesbaar.

Reply:
Check, ik ga ervoor kiezen om allen de `getDate()` te checken zonder de empty string